Mathlib/Analysis/Normed/Group/Constructions.lean

```diff
@@ -91,6 +93,32 @@ instance normedCommGroup [NormedCommGroup E] : NormedCommGroup (ULift E) :=
 
 end ULift
 
+
+/-! ### SeparationQuotient -/
```

Comment:
I'm putting this in this file, since the `MetricSpace (SeparationQuotient _)` instance is defined next to the `MetricSpace (Additive _)` instance, and this is consistent.

Reply:
I think I would rather put it in `Analysis.Normed.Group.Uniform`, just for the sake of keeping `NormedField` a "cheap" import.

Reply:
Sounds reasonable, done.